Engines/Wine/Verbs/gallium9/script.js

```diff
@@ -53,14 +54,14 @@ Wine.prototype.gallium9 = function (gallium9Version) {
 
 /**
  * Verb to install Gallium 9 Standalone
-*/
+ */
 var verbImplementation = {
     install: function (container) {
-        var wine = new Wine();
+        let wine = new Wine();
```

Comment:
Why `let`? `wine` is also a constant you don't replace it later on :)

When thinking about this it makes sense to move the `const wine = new Wine()` line between the current lines 64 and 65. This change would group all the wine related commands together.